### DISCUSSION TITLE
#1

### DISCUSSION THREAD

**Discussion by TarunavBA** [LazyBugged/lazy.bugged#5] [Discuss]
Hey @BaseMax can you improve the app and the mkdocs documentation? Don't forget to create a PR and commit your changes in that branch.

**Comment by BaseMax** [comment]
- **httos** is wrong, use HTTPS
- you need to rename repository name to name of org to keep them same. (**LazyBugged**)

Please change ` httos://LazyBugged.github.io/lazy.bugged ` to `https://lazybugged.github.io/LazyBugged/`

**Reply by TarunavBA** [reply]
@BaseMax done :shipit:

**Reply by TarunavBA** [reply]
@BaseMax sorry it was a typo 🤷‍♂️

**Comment by TarunavBA** [comment]
@BaseMax You can refer to the tutorial in the official flask website in how to make a blog.
